jobbergate-api/jobbergate_api/config.py

```diff
@@ -32,13 +32,19 @@ class Settings(BaseSettings):
 
     LOG_LEVEL: LogLevelEnum = LogLevelEnum.INFO
 
-    # Database settings
-    DATABASE_HOST: Optional[str]
-    DATABASE_USER: Optional[str]
-    DATABASE_PSWD: Optional[str]
-    DATABASE_NAME: Optional[str]
+    # Database settings  # Default to values from docker-compose.yml
+    DATABASE_HOST: Optional[str] = "db"
+    DATABASE_USER: Optional[str] = "jobbergate"
+    DATABASE_PSWD: Optional[str] = "local-pswd"
+    DATABASE_NAME: Optional[str] = "jobbergate"
     DATABASE_PORT: Optional[int] = 5432
-    DATABASE_URL: Optional[str]
+
+    # Test database settings
+    TEST_DATABASE_HOST: Optional[str] = "localhost"
+    TEST_DATABASE_USER: Optional[str] = "test"
+    TEST_DATABASE_PSWD: Optional[str] = "test-pswd"
+    TEST_DATABASE_NAME: Optional[str] = "test-jobbergate"
+    TEST_DATABASE_PORT: Optional[int] = 5433
```

Comment:
we could set this stuff in the docker-compose file and remove complexity from the code itself

Reply:
These are set in the docker-compose. Having default values here doesn't add complexity in my opinion.

Reply:
I am lost to be honest. Some are set in the compose, some are not...

Reply:
The docker-compose.yaml needs some cleanup. Removing defaults here isn't a great approach, though, because it means you have to set environment variables before you can do anything with the project. By setting defaults on the database variables, I can spin up local and test databases with the dev-tools without having to set environment variables at all.